oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/ServiceRelation.java

```diff
@@ -61,11 +62,17 @@ public String getEntityId() {
     @Setter
     private String sourceServiceInstanceName;
     @Getter
+    @Setter
+    private Layer sourceLayer;
+    @Getter
     private String destServiceId;
     @Getter
     @Setter
     @ScopeDefaultColumn.DefinedByField(columnName = "dest_name", requireDynamicActive = true)
     private String destServiceName;
+    @Getter
+    @Setter
+    private Layer destLayer;
```

Comment:
Should relation needs layer?  if so the relations here: https://github.com/apache/skywalking/blob/master/oap-server/server-receiver-plugin/skywalking-mesh-receiver-plugin/src/main/java/org/apache/skywalking/aop/server/receiver/mesh/TelemetryDataDispatcher.java
are needs to set to `Layer.mesh` @wu-sheng

Reply:
Make sense. Is MAL having required change too?